### ISSUE TITLE
损失函数问题

### ISSUE THREAD

**Issue by mlpod** [yongliang-wu/DFT#2]
dft 的损失函数可以更简洁一些：
loss_fct = nn.CrossEntropyLoss(reduction="none")
sft_loss = loss_fct(shift_logits, shift_labels)
dft_loss=sft_loss*torch.exp(-sft_loss).detach()

我在代码中打印了dft_loss和sft_loss，sft_loss直接起飞了，这个正常吗？

<img width="1898" height="568" alt="Image" src="https://github.com/user-attachments/assets/5fbe3fcd-edef-47ee-8e1f-dd26160e2e82" />

**Comment by yongliang-wu** [comment]
sft_loss起飞是正常的，不过后期会稳定在一定的数值上，你可以多跑一会看看，我观察到在数学上面，大概会稳定在2.5-3.5之间
